charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,65 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Data Processing Service is a service that executes data processing pipelines on DCP data in the Data Storage Service. It can be used in any instance of the Data Coordination Platform (DCP). 
+
+## Objectives
+
+The Data Processing Service aims to develop the infrastructure to run pipelines reliably and at-scale for the DCP. The team working on the service will also operate the infrastructure for the Human Cell Atlas. 
```

Comment:
_operate the infrastructure_ -> _operate the *pipeline* infrastructure_ 

Editorial:
_The Data Processing Service ~aims to~ develops the infrastructure to run pipelines reliably and at-scale for the DCP. The team ~working on~ implementing the service ~will~ also operate**s** the pipeline infrastructure for the Human Cell Atlas._